RGBmatrixPanel.cpp

```diff
@@ -31,10 +31,65 @@ lower CPU utilization:
 Written by Limor Fried/Ladyada & Phil Burgess/PaintYourDragon for
 Adafruit Industries.
 BSD license, all text above must be included in any redistribution.
+
+    ****************
+
+Support for Feather M0 contributed by ee-quipment.com
+
+To support the Feather M0, direct writes to AVR ports have been replaced with
+direct writes to Zero ports that work on the Feather M0. The pinouts are fixed
+and do not conflict with the SPI or I2C ports. The LED_BUILTIN port is used however.
+
+Panel Pins:      OE   LAT        B2    G2    R2    B1    G1    R1       CLK     E         D         C     B         A
+Connector Pins:  15    14         7     6     5     3     2     1        13     8        12        11    10         9
+Feather Pins:     7     6        20    24    22    25    23    19        14    15        21         9     8         5
+Port Pins:      PB09  PB08  ..  PA20  PA19  PA18  PA17  PA16  PA15  ..  PA11  PA10  ..  PA07  ..  PA05  PA04  ..  PA02
+Arduino IDE:     16    15         6    12    10    13    11     5         0     1         9        18    17        14
+
+Schematics and a board layout are available at https://github.com/ee-quipment/RGB-matrix-Panel-Zero
+A bare board can be ordered from OshPark at https://oshpark.com/shared_projects/1QNzmjwa
+
+To set up interrupts a third-party library is used:
+#include  <avdweb_SAMDtimer.h>  //http://www.avdweb.nl/arduino/libraries/samd21-timer.html
+
 */
 
 #include "RGBmatrixPanel.h"
-#include "gamma.h"
```

Comment:
In your PR, you mention there is a conflict with math.h.
Is that a problem with a regular arduino mega build too? (you can do a dummy build even if you don't have the board), or just your board?
If the conflict is the gamma name, why not rename it in gamma.h?
In other words, why copy the gamma.h definition in this file just to rename the array?

Reply:
So, I just took arduino 1.8.5, the latest released version, selected arduino mega, took PanelGFXDemo, built it, and it built fine, no issues at all.
You probably have 3rd party libraries that are conflicting?
I also looked at my build output, there is no math library that gets pulled in.

So, I think you'll have to first resolve why the build is broken for you on arduino mega (use a clean tree and hide/remove libraries you may have added).
Once that's figured out, you can likely revert the gamma portion of your patch, unless it is actually necessary for feather, but even then, you'd want to rename the variable in gamma.h, not just copy its content to then rename it.

Reply:
I appreciate your looking into the gamma conflict further. That's an interesting data point and I don't really know what is going on. But is does seem like bad form to conflict with the standard library, doesn't it? Anyway, if you or anyone else would care to modify the code and restore it or change it or whatever, I have no objection. I'm just putting this out there since I needed it and thought others using a Feather M0 might also find it useful.

Reply:
Just as a heads up, it's not quite how it works. Adafruit is super busy and barely has time to review patches. They don't have time to modify or improve patches.
Ultimately it's on you to give the minimally intrusive patch that will apply cleanly and add the functionality you mentioned. 
If you post your gamma build problem with math.h, I might be able to help you with it.
If you leave the patch as is, it's very unlikely to get modified and merged by the adafruit folks although someone random might look for that support and find your patch here.

Reply:
Thanks again for your offer.

Reply:
By the way, your latest update to not include gamma.h is going the wrong way :)
Assuming there is an unsolvable name conflict, all you need to do is rename the variable in gamma.h I believe :)